src/frontend/src/pages/search/results/groupby/GroupBy.vue

```diff
@@ -42,7 +42,7 @@
 						data-width="auto"
 						data-menu-width="auto"
 						hideEmpty />
-				<i18n path="results.groupBy.iWantToGroupOnAnnotation" tag="div">
+				<i18n :path="'results.groupBy.' + (contextValue.startsWith(OPT_PREFIX_SPAN_ATTRIBUTE) ? 'iWantToGroupOnNoAnnotation' : 'iWantToGroupOnAnnotation')" tag="div">
```

Comment:
Translation keys beter samenhouden ipv concatten/interpolaten, anders wordt het onmogelijk om terug te vinden waar/of keys gebruikt worden.

Reply:
:+1: